hazelcast/src/main/java/com/hazelcast/map/IMap.java

```diff
@@ -2978,4 +2980,29 @@ default void addIndex(IndexType type, String... attributes) {
      */
     V computeIfAbsent(@Nonnull K key, @Nonnull Function<? super K, ? extends V> mappingFunction);
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p> </p>
+     * <p>
+     *     If the supplied {@code action} is a lambda, anonymous class or an inner class,
+     *     it would be executed locally. Same would happen if it is not serializable.
```

Comment:
You may also add that the implementation will pull in all of the map data onto the local member in this case because it uses `IMap#entrySet()` to iterate over the entries.